finality-aleph/src/testing/network.rs

```diff
@@ -1,625 +1,627 @@
-    for node_id in 1..NODES_N {
-        let data_1 = MockData::new((node_id - 1) as u32, 1);
-        let data_2 = MockData::new(node_id as u32, 1);
-
-        expected_data.insert((
-            data_1.clone(),
-            SessionId(session_id_1),
-            test_data.authorities[node_id].peer_id(),
-        ));
-        data_network_1
-            .send(data_1, Recipient::Node(NodeIndex(node_id)))
-            .expect("Should send");
-
-        expected_data.insert((
-            data_2.clone(),
-            SessionId(session_id_2),
-            test_data.authorities[node_id].peer_id(),
-        ));
-        data_network_2
-            .send(data_2, Recipient::Node(NodeIndex(node_id)))
-            .expect("Should send");
-    }
-
-    let mut sent_data = HashSet::new();
-    while sent_data.len() < 2 * (NODES_N - 1) {
-        if let Some((DataInSession { data, session_id }, peer_id)) =
-            test_data.validator_network.send.next().await
-        {
-            sent_data.insert((data, session_id, peer_id));
-        }
-    }
-
-    assert_eq!(sent_data, expected_data);
-    test_data.cleanup().await;
-
-    assert_eq!(
-        timeout(DEFAULT_TIMEOUT, data_network_1.next()).await,
-        Ok(None)
-    );
-    assert_eq!(
-        timeout(DEFAULT_TIMEOUT, data_network_2.next()).await,
-        Ok(None)
-    );
-}
-
-#[tokio::test]
-async fn test_broadcasts_data_to_correct_session() {
-    let session_id_1 = 42;
-    let session_id_2 = 43;
-    let mut test_data = prepare_one_session_test_data().await;
-    let mut data_network_1 = test_data.start_session(session_id_1).await;
-    let mut data_network_2 = test_data.start_session(session_id_2).await;
-
-    let data_1 = MockData::new(43, 3);
-    let data_2 = MockData::new(44, 3);
-    data_network_1
-        .send(data_1.clone(), Recipient::Everyone)
-        .expect("Should send");
-    data_network_2
-        .send(data_2.clone(), Recipient::Everyone)
-        .expect("Should send");
-
-    let mut expected_data = HashSet::new();
-    for authority in test_data.authorities.iter().skip(1) {
-        expected_data.insert((data_1.clone(), SessionId(session_id_1), authority.peer_id()));
-        expected_data.insert((data_2.clone(), SessionId(session_id_2), authority.peer_id()));
-    }
-
-    let mut sent_data = HashSet::new();
-    while sent_data.len() < 2 * (NODES_N - 1) {
-        if let Some((DataInSession { data, session_id }, peer_id)) =
-            test_data.validator_network.send.next().await
-        {
-            sent_data.insert((data, session_id, peer_id));
-        }
-    }
-
-    assert_eq!(sent_data, expected_data);
-
-    test_data.cleanup().await;
-
-    assert_eq!(
-        timeout(DEFAULT_TIMEOUT, data_network_1.next()).await,
-        Ok(None)
-    );
-    assert_eq!(
-        timeout(DEFAULT_TIMEOUT, data_network_2.next()).await,
-        Ok(None)
-    );
-}
+// TODO: integrate new Substrate network API properly
+
+// use std::{
```

Comment:
Commented code...

Just delete the file and leave a TODO comment in the corresponding `mod.rs` file.

Reply:
removed, no TODO, will remember to add tests in a separate PR